uvm/servlets/admin/config/network/Interface.js

```diff
@@ -804,6 +794,40 @@ Ext.define('Ung.config.network.Interface', {
                 maxLength: 63,
                 minLength: 8,
                 maskRe: /[a-zA-Z0-9~@#%_=,\!\-\/\?\(\)\[\]\\\^\$\+\*\.\|]/
+            }, {
+                xtype: 'combo',
+                fieldLabel: 'Regulatory Country'.t(),
+                width: 475,
+                bind: {
+                    value: '{intf.wirelessCountryCode}',
+                    store: '{wirelessCountryList}',
+                    hidden: '{intf.wirelessMode === "CLIENT"}'
+                },
+                editable: false,
+                queryMode: 'local',
+                listeners: {
+                    afterrender: function(){
+                        // Default value for country code is empty so it can be populated from cmd.
+                        // If empty now, get the current country code from the driver.
+                        var me = this;
+                        var vm = me.up('window').getViewModel();
+                        if(!vm.get('intf').get('wirelessCountryCode')){
+                            // Directly get the value since the bind won't be complete by the time we get here.
+                            Ext.Deferred.sequence([
+                                Rpc.asyncPromise('rpc.networkManager.getWirelessRegulatoryCountryCode', vm.get("intf").get("systemDev"))
+                            ],this)
+                            .then(function(result) {
+                                me.setValue(result[0]);
```

Comment:
What if get_regulatory_country_code fails to grab either the physical name (setting the region normally) or global? None would be returned by Python, so presumably this result[0] call would attempt to get the first result of a null type?